pkgs/top-level/all-packages.nix

```diff
@@ -2857,9 +2857,6 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security AppKit CoreServices;
     electron = electron_13;
   };
-  element-desktop-wayland = element-desktop.override {
```

Comment:
I think we should add aliases for those.

Reply:
If it hasn't been in a stable release then it doesn't need and alias so check that first

Reply:
Well we probably can't avoid it getting released with 21.11 so that will be a stable release.

The alias should just be a wrapper that sets the environment variable, no biggie.